### DISCUSSION TITLE
Weird error

### DISCUSSION THREAD

**Discussion by Headcold** [ABJC/ABJC-tvOS#33] [General]
Hello

I installed the beta today on my appletv and after I enter address, username and password I get an error.

Alerts.auth.title
Alerts.auth.failed
Bottoms.ok.  - is the name of the button on main screen

Is https:\\ supported?

**Comment by NoahKamara** [comment]
Hey there,
sorry for the very late reply, I didn't even know I had the discussions section enabled :O 
the bug should be resolved by now
